README.markdown

```diff
@@ -1879,6 +1880,57 @@ cipher usage:
 
 [Back to TOC](#table-of-contents)
 
+lua-leak
+---------------------------
+This script collects all memory not freed during the tracing period and corresponding backtraces,
+which supports both compiled code and interpretered lua code.
+
+Note that it could not check memory allocated via ffi call. If lua code is clean,
+you could turn to use `sample-bt-leaks.sxx` to check possible memory leak outside lua code.
+
+Because the memory leak involves a lot of allocation, to avoid memory overflow of statistics array,
+it would print report and restart collection each time when the number of allocations not freed exceeds `100000`.
+
+Here is an example from one [apisix issue](https://github.com/apache/apisix/issues/7949):
+
```

Comment:
There are a lot of comments under this issue, can you give a specific comment link?